tests/data/dataset_test.py

```diff
@@ -14,16 +14,16 @@
 class TestDataset(AllenNlpTestCase):
     def setUp(self):
         self.vocab = Vocabulary()
-        self.vocab.add_token_to_namespace("this", "words")
-        self.vocab.add_token_to_namespace("is", "words")
-        self.vocab.add_token_to_namespace("a", "words")
-        self.vocab.add_token_to_namespace("sentence", "words")
-        self.vocab.add_token_to_namespace(".", "words")
+        self.vocab.add_token_to_namespace("this")
```

Comment:
Unrelated to this PR - I see this line and I think "add_token_to_namespace" is too verbose.  Would it be problematic to just change this to `add_token`, using the `namespace=''` argument when necessary?